third-party/cmake/FetchGlog.cmake

```diff
@@ -12,22 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set(GLOG_URL https://github.com/google/glog/archive/refs/tags/v0.4.0.tar.gz)
+set(GLOG_URL https://github.com/google/glog/archive/refs/tags/v0.6.0.tar.gz)
 
 message(STATUS "build glog from ${GLOG_URL}")
 
 find_program(MAKE_EXE NAMES gmake nmake make)
 ExternalProject_Add(
   glog
   URL ${GLOG_URL}
-  URL_HASH SHA256=f28359aeba12f30d73d9e4711ef356dc842886968112162bc73002645139c39c
+  URL_HASH SHA256=8a83bf982f37bb70825df71a9709fa90ea9f4447fb3c099e1d720a439d88bad6
   PREFIX ${DEPS_BUILD_DIR}
   DOWNLOAD_DIR ${DEPS_DOWNLOAD_DIR}/glog
   INSTALL_DIR ${DEPS_INSTALL_DIR}
   DEPENDS gflags
   BUILD_IN_SOURCE TRUE
-  CONFIGURE_COMMAND
-    ./autogen.sh
-    COMMAND CXXFLAGS=-fPIC ./configure --prefix=<INSTALL_DIR> --enable-shared=no --with-gflags=<INSTALL_DIR>
-  BUILD_COMMAND ${MAKE_EXE}
-  INSTALL_COMMAND ${MAKE_EXE} install)
+  CONFIGURE_COMMAND ${CMAKE_COMMAND} -H<SOURCE_DIR> -B <BINARY_DIR> -DWITH_GLOG=ON -DCMAKE_CXX_FLAGS=-fPIC
```

Comment:
although its the same, `${DPES_INSTALL_DIR}` -> `<INSTALL_DIR>` looks more unified.